crates/e2e/tests/e2e/partially_fillable_observed_score.rs

```diff
@@ -156,6 +157,10 @@ async fn test(web3: Web3) {
 
     tracing::info!(?trades, ?competition_0, ?competition_1);
 
+    let latest_competition = services.get_latest_solver_competition().await.unwrap();
+    tracing::info!(?latest_competition);
+    assert_eq!(latest_competition.auction_id, auction_with_id.id);
```

Comment:
The latest competition JSON should probably be identical to `competition_1` so best to check that for equality entirely.

Reply:
done